monitoring_versioned_docs/version-22.10/installation/setup-central-poller/import-virtual-appliance.md

```diff
@@ -8,7 +8,7 @@ title: How to import a Virtual appliance
 **i-Vertix** support provides a ready-to-use virtual machine.
 This virtual machine is available in **OVA** format for VMware environments.
 
-It's based on the **Linux CentOS 7** (and all required packages) operating system and includes a i-Vertix installation that allows you to easily start your first monitoring.
+It's based on the **Oracle Linux 8** (and all required packages) operating system and includes a i-Vertix installation that allows you to easily start your first monitoring.
```

Comment:
Perfavore in fondo di questo capitolo cambiare il screenshot con uno recente i-Vertix4

Inoltre per i-Vertix4 mettere specificare come prerequisito VMware 6.7 (i-Vertix3 VMware 6.5) o meglio ancora mettere il link alla pagina Virtualization compatibility

Reply:
ok fatto